CorsixTH/Lua/world.lua

```diff
@@ -2713,6 +2768,10 @@ function World:afterLoad(old, new)
     self.wall_set_by_block_id = nil
     self.wall_dir_by_block_id = nil
   end
+  if old < 172 then
+    self.wall_types = self.app.wall_types
+    self:initWallTypes()
+  end
```

Comment:
Arguably deleting the wall variables in one version then readding them in the next is not optimal or sensible now.
It goes against convention but can't we modify `if old < 171 then` code so this weirdness doesn't occur moving forward?

Else, if you're considering a new class or something I'm not sure what's the best course of action? Scuttle this PRs current code perhaps and bring in the new class as the 'fix'?

Reply:
I am not considering anything currently, tbh I am quite done with fighting the language and its lack of adequate tools.